### PR TITLE
expect caseId in the root of the response

### PR DESCRIPTION
**Pull request by rorticus0502**

### CODE REVIEW

File: app/services/ClaimService.scala [hunk 1]
```diff
@@ -42,8 +42,8 @@ class ClaimService @Inject() (
     maybeRegistrationRequest match {
       case Some(value) =>
         connector.submitClaim(value, correlationId(hc)).map { clientClaimResponse =>
-          if (clientClaimResponse.result.map(_.caseId).nonEmpty)
-            clientClaimResponse.result.map(_.caseId).get
+          if (clientClaimResponse.caseId.nonEmpty)
+            clientClaimResponse.caseId.get
```
Comment (SteveSugden) [inline app/services/ClaimService.scala:46]: I know this was already like this, but can this be refactored to get rid of the "if" and "get" statements?

File: app/services/ClaimService.scala [hunk 2]
```diff
@@ -69,8 +69,8 @@ class ClaimService @Inject() (
     maybeAmendRequest match {
       case Some(value) =>
         connector.submitAmendClaim(value, correlationId(hc)).map { clientClaimResponse =>
-          if (clientClaimResponse.result.map(_.caseId).nonEmpty)
-            clientClaimResponse.result.map(_.caseId).get
+          if (clientClaimResponse.caseId.nonEmpty)
+            clientClaimResponse.caseId.get
```
Comment (SteveSugden) [inline app/services/ClaimService.scala:73]: I know this was already like this, but can this be refactored to get rid of the "if" and "get" statements?

**Review by SteveSugden — COMMENTED** [review]
Looks good but comment for further credit

File: app/services/ClaimService.scala (hunk 1 above)
Comment (rorticus0502) [reply app/services/ClaimService.scala:46]: Yes, Of course. I wanted you to see it exactly as it was, then you could decide whether you preferred to refactor slightly or go for minimal change. I will do it now.

File: app/services/ClaimService.scala (hunk 2 above)
Comment (rorticus0502) [reply app/services/ClaimService.scala:73]: on it

**Review by SteveSugden — APPROVED** [review]

**Pull request merged**